### PR TITLE
Add leaflet examples

### PR DESCRIPTION
**Pull request by Pessimistress**
Get-started examples using deck.gl-leaflet by @zakjan

### CODE REVIEW

File: examples/get-started/pure-js/leaflet/app.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import * as Leaflet from 'leaflet';
```
Comment (kylebarron) [inline examples/get-started/pure-js/leaflet/app.js:1]: It seems canonical to import leaflet as `L` (or at least that's how it's exposed when included as a script, I think); don't have a strong preference whether we follow that.

**Review by kylebarron — APPROVED** [review]

**Comment by zakjan** [comment]
@Pessimistress This is great! Did you have any issues with using deck.gl-leaflet? I know it's missing picking support so far.

I'm going to explore a possibility to move deck.gl-leaflet to a community plugin repo next week (after getting back from holiday). // cc @ibgreen 

Also it's interesting for me to compare the differences and API of existing integration plugins. It might be useful to unify them in a long run.

**Comment by Pessimistress** [comment]
> Also it's interesting for me to compare the differences and API of existing integration plugins. Do you think it might be useful to unify them in a long run?

Definitely. I find myself writing very similar bootstrapping code for other integration project as well.

> it's missing picking support so far.

So I have noticed. Let me know if you need help implementing it!

**Comment by Pessimistress** [comment]
@zakjan A small issue I noticed is this line in deck.gl-leaflet:

```js
import { Deck } from 'deck.gl';
```

It's better to use

```js
import { Deck } from '@deck.gl/core';
```

as well as specifying `@deck.gl/core` as peer dependency. Not all projects install the deck.gl master module.

**Comment by zakjan** [comment]
Thanks, right I forgot about this. I published a new version with this change.

**Pull request merged**